packages/foreman/nodejs-node-gyp/node-gyp-node-dirs.patch

```diff
@@ -0,0 +1,46 @@
+diff --git a/lib/configure.js b/lib/configure.js
+index 2ff476d..7a69e1c 100644
+--- a/lib/configure.js
++++ b/lib/configure.js
+@@ -62,7 +62,9 @@ function configure (gyp, argv, callback) {
+         log.verbose('get node dir', 'compiling against --target node version: %s', release.version)
+       } else {
+         // if no --target was specified then use the current host node version
++        // if no --target was specified then use RPM installed headers
+         log.verbose('get node dir', 'no --target version specified, falling back to host node version: %s', release.version)
++        log.verbose('get node dir', 'no --target version specified, falling back to RPM installed headers')
+       }
+
+       if (!release.semver) {
+@@ -75,7 +77,7 @@ function configure (gyp, argv, callback) {
+       gyp.commands.install([ release.version ], function (err, version) {
+         if (err) return callback(err)
+         log.verbose('get node dir', 'target node version installed:', release.versionDir)
+-        nodeDir = path.resolve(gyp.devDir, release.versionDir)
++        nodeDir = '/opt/rh/rh-nodejs10/root/usr/share/node'
```

Comment:
Is there a way we could determine this automatically in case we change the SCL? Looks like there's a `nodeDir` parameter and can't we pass that in?

Reply:
This file actually should be in the node-gyp PR. Will fix

Reply:
As to the question, I honestly don't know. And I on the fence how much time its worth spending diving into this code vs. taking this simple patch that works.